src/vl-map-draw-action.js

```diff
@@ -22,4 +24,38 @@ export class VlMapDrawAction extends VlMapLayerAction {
   onDraw(callback) {
     this.__callback = callback;
   }
+
+  get __drawOptions() {
+    if (this.dataset.vlSnapping !== undefined) {
+      const snappingLayers = this.__snappingLayers;
+      if (snappingLayers.length == 0) {
+        return {snapping: true};
+      } else {
+        return {
+          snapping: {
+            layer: this.__createSnappingLayer(),
+            pixelTolerance: this.dataset.vlSnappingPixelTolerance || 10,
+            node: false,
+            vertex: false,
+          },
+        };
+      }
+    } else {
+      return {};
+    }
+  }
+
+  __createSnappingLayer() {
+    const snappingLayer = new VlCompositeVectorLayer(this.__snappingLayers.map((layer) => layer._layer), {});
```

Comment:
De `{}` legt bloot dat de `VlCompositeVectorLayer` constructie een 2e argument verwacht. Beter zou geweest zijn dat het 2e argument optioneel is en misschien moeten we dat nog uitvoeren in een 4.0.1 release.

Reply:
Gij via de master of ik via ne PR?

Reply:
Ik zal dat via een PR doen en hier 4.0.1 gebruiken, dan zijn er geen afhankelijkheden.

Reply:
Alternatief is hier toch een 'dummy' titel zetten. Verwarring voor de eindgebruiker is niet echt mogelijk omdat die laag niet zal getoond worden in de layer switcher ...

Reply:
Zie PR [#51](https://github.com/milieuinfo/vl-mapactions/pull/51).

Reply:
> Alternatief is hier toch een 'dummy' titel zetten. Verwarring voor de eindgebruiker is niet echt mogelijk omdat die laag niet zal getoond worden in de layer switcher ...

Dan zou ik eerder een default voorzien op `vl-mapactions` niveau, maar voorlopig dus optioneel gemaakt via PR #51.